detection_rules/rule_loader.py

```diff
@@ -17,17 +17,17 @@
 from marshmallow.exceptions import ValidationError
 
 from . import utils
+from .config import parse_rules_config
 from .mappings import RtaMappings
 from .rule import (
     DeprecatedRule, DeprecatedRuleContents, DictRule, TOMLRule, TOMLRuleContents
 )
 from .schemas import definitions
 from .utils import cached, get_path
 
-DEFAULT_PREBUILT_RULES_DIR = Path(get_path("rules"))
-DEFAULT_PREBUILT_BBR_DIR = Path(get_path("rules_building_block"))
-DEFAULT_PREBUILT_DEPRECATED_DIR = DEFAULT_PREBUILT_RULES_DIR / '_deprecated'
-DEFAULT_PREBUILT_RTA_DIR = get_path("rta")
+RULES_CONFIG = parse_rules_config()
```

Comment:
I dont see this used in this file

Reply:
It is imported in other files like devtools.py and test_all_rules.py

Reply:
It is also used on the following line (line 29)

Reply:
lol yup for some reason I didn't see this before!